### ISSUE TITLE
[BUG] GitHub Action fails after ColorLib change

### ISSUE THREAD

**Issue by sneak-it** [surftimer/Surftimer-olokos#214]
**Describe the bug**
The GH actions fail after ColorGen was moved to its own repo: https://github.com/c0rp3n/colorlib-gen

**Expected behavior**
The obvious

**Screenshots**

**Plugin List**

**Additional context**

@Bara - if you don't mind taking a look at this, not sure if it's a quick fix or not, looks like the ColorGen structure changed quite a bit.

**Comment by Bara** [comment]
```yml
      - name: Set up Python
        uses: actions/setup-python@v2
        with:
          python-version: '3.x'
      
      - name: Install dependencies
        run: python3 -m pip install --upgrade pip setuptools wheel

      - name: Download colorlib-gen
        run: |
          cd tools
          git clone https://github.com/c0rp3n/colorlib-gen.git
      
      - name: Install colorlib-gen
        run: |
          cd tools/colorlib-gen
          pip install -r ./requirements.txt
          python3 ./setup.py install

      - name: Generate colorlib_map.inc
        run: colorlib_gen -r --config "./ttt_colors.yaml" "../addons/sourcemod/scripting/include/colorlib_map.inc"
        working-directory: ./tools`
```
Is what we use in TTT since yesterday.

**Comment by sneak-it** [comment]
Resolved with #215
